.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+    pull_request: {}
+    push:
+      branches:
+        - master
+        - 1.*
```

```suggestion
        - *.*.x
```

---

Beware if you start the pattern with a star: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

---

I would not do that, yeah. 2.0 is going to be a big change anyways, so it is okay to add a line to the CI file by the time it is happening :)

---

how about this safer pattern:
```suggestion
        - [0-9]+.[0-9]+.x
```

---

Last suggestion (what I'm doing in core's latest PR, turns out the quotes are necessary here too):
```suggestion
        - "[0-9].[0-9]+.x"
```